huaweicloud/services/organizations/resource_huaweicloud_organizations_account_associate.go

```diff
@@ -68,12 +68,13 @@ func ResourceAccountAssociate() *schema.Resource {
 
 func resourceAccountAssociateCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
 
 	// createAccountAssociate: create Organizations account associate
 	var (
 		createAccountAssociateProduct = "organizations"
 	)
-	createAccountAssociateClient, err := cfg.NewServiceClient(createAccountAssociateProduct, "")
+	createAccountAssociateClient, err := cfg.NewServiceClient(createAccountAssociateProduct, region)
```

Comment:
the **read** and **delete** methods are missing

Reply:
done